ovirt/provider.go

```diff
@@ -66,5 +72,6 @@ func ConfigureProvider(d *schema.ResourceData) (interface{}, error) {
 		Username(d.Get("username").(string)).
 		Password(d.Get("password").(string)).
 		Insecure(true).
+		Headers(d.Get("headers").(map[string]string)).
```

Comment:
If the optional `headers` has no default value/function, we need to check if it's a unset value.

Reply:
See above, I added a `Default` to the schema.